applications/directory/src/components/ProfileDetails.js

```diff
@@ -148,36 +169,54 @@ const ProfileDetails = ({ artist, isLoggedIn = false }) => {
 
         {social.length > 0 &&
           social.map(({ name, url }, index) => (
-            <Box key={index} sx={{ mb: '10px' }}>
+            <Box key={index} sx={{ marginTop: '1rem' }}>
               <Box
                 sx={{
                   display: 'flex',
                   alignItems: 'center',
-                  '& span': {
-                    fontSize: '12px',
-                    fontWeight: '500',
-                    fontStretch: 'normal',
-                    fontStyle: 'normal',
-                    lineHeight: '1.33',
-                    letterSpacing: '0.4px',
-                    mt: '1px',
-                    color: 'rgba(0, 0, 0, 0.5)',
-                    '&:first-letter': {
-                      textTransform: 'capitalize'
-                    },
+                  justifyContent: ['center', 'flex-start']
+                }}
+              >
+                <Box
+                  component="span"
+                  sx={{
                     '& svg': {
                       fontSize: 20,
                       fontWeight: 'bold',
                       marginRight: '5px',
                       color: 'primary.main'
                     }
+                  }}
+                >
+                  {socialIcons[name]}
+                </Box>
+                <Box
+                  component="span"
+                  sx={{
+                    fontSize: ['0.75rem', '0.75rem', '0.75rem', '0.75rem'],
+                    fontWeight: '500',
+                    lineHeight: '1.33',
+                    letterSpacing: '0.4px',
+                    color: 'text.secondary',
+                    textTransform: 'capitalize'
+                  }}
+                >
+                  {name}
+                </Box>
+              </Box>
+              <Box
+                sx={{
+                  '& a': {
```

Comment:
Can we move this `& a` styling to the `Link` component below? To support this, the `Link` component would probably need to change link this:

```javascript
<Box as="a" href={href} target="_blank" rel="noopener noreferrer" {...props}>
  {children}
</Box>
```

Reply:
done